src/components/configuration/Configuration.ts

```diff
@@ -96,7 +96,7 @@ export class Configuration {
             undefined
         );
 
-        configuration.updatePathToRlsExecutableSpecifiedByUser();
+        configuration.updateUserRlsPath();
```

Comment:
All names have ambiguity (for example, "update" is quite ambiguous here), and basically I ends up reading the code. That's why IDE provides the jump feature.